tests/sentry/api/endpoints/test_organization_repositories.py

```diff
@@ -48,6 +48,32 @@ def test_get_integration_repository(self):
         assert first_row["provider"] == {"id": "dummy", "name": "Example"}
         assert first_row["externalSlug"] == str(repo.external_id)
 
+    def test_get_active_repos(self):
+        repo = Repository.objects.create(
+            name="getsentry/example",
+            organization_id=self.org.id,
+            external_id=12345,
+            provider="dummy",
+            config={"name": "getsentry/example"},
+        )
+        Repository.objects.create(
+            name="getsentry/sentry",
+            organization_id=self.org.id,
+            external_id=54321,
+            provider="dummy",
+            config={"name": "getsentry/sentry"},
+            status=ObjectStatus.HIDDEN,
```

Comment:
so `status` is an optional attr?

Is there already a test for multiple active repos?

Reply:
no but i can add one

Reply:
the default for status is `ObjectStatus.ACTIVE`